tabReplace.c

```diff
@@ -2,66 +2,100 @@
 replaces TAB characters in user input with spaces
```

Comment:
nit: These 2 files are practically indentical except for the input parameter for the shared `process` helper function. They could be combined into a single file that asks for the tab-to-space or space-to-tab conversion upfront
```
tabReplace.c
-------------
process(tabSize, drop, inputFile, outputFile, '\t');

spaceReplace.c
---------------
process(tabSize, drop, inputFile, outputFile, ' ');
```

Reply:
Good point, will address when handling #3 for full optional replacement support. Currently replacement option creates a `tmp.txt` for the user to replace the input file themselves since overwriting the existing file could have irreversible consequences